src/components/PlaceSearch.vue

```diff
@@ -1,69 +1,72 @@
 <template>
-  <v-sheet class="searchContainer">
-    <v-autocomplete
-      v-model="model"
-      v-model:search="search"
-      auto-select-first
-      class="rounded"
-      clearable
-      :custom-filter="filter"
-      density="compact"
-      hide-details
-      hide-no-data
-      item-title="properties.name"
-      :items="items"
-      label="Ort, Adresse, Flurname,..."
-      :loading="!!abortController"
-      :prepend-inner-icon="mdiMagnify"
-      return-object
-      single-line
-      variant="outlined"
-      @blur="handleInfoVisibility(false)"
-      @click:clear="clear"
-      @focus="handleInfoVisibility(true)"
-    >
-      <template #item="{ props, item }">
-        <v-list-item
-          v-bind="props"
-          :subtitle="item.raw.type"
-          :title="item.raw.properties.name"
-        ></v-list-item>
-      </template>
-    </v-autocomplete>
-    <v-expand-transition>
-      <v-card v-show="showInfo">
-        <v-card-title>
-          <v-icon :icon="mdiInformationOutline" size="small" />
-          Ortssuche
-        </v-card-title>
-        <v-card-subtitle>
-          Die Suche des
-          <a href="https://kataster.bev.gv.at" target="_blank"
-            >Österreichischen Katasters</a
-          ><br />
-          Suche nach Orten, Adressen, und mehr
-        </v-card-subtitle>
-        <v-card-text>
-          <v-timeline align="start" density="compact" line-thickness="0">
-            <v-timeline-item
-              v-for="(helpItem, key) in helpItems"
-              :key="key"
-              density="compact"
-              dot-color="success"
-              :icon="mdiTextSearchVariant"
-            >
-              <div>
-                <div class="font-weight-normal">
-                  <strong> {{ helpItem.name }}</strong>
-                </div>
-                {{ helpItem.example }}
+  <v-autocomplete
+    ref="autocompleteRef"
+    v-model="model"
+    v-model:search="search"
+    auto-select-first
+    class="rounded"
+    clearable
+    :custom-filter="filter"
+    density="compact"
+    hide-details
+    hide-no-data
+    item-title="properties.name"
+    :items="items"
+    label="Ort, Adresse, Flurname,..."
+    :loading="!!abortController"
+    min-width="280px"
```

Comment:
Aren't properties like `min-width` supposed to be numeric values?
```suggestion
    min-width="280"
```